### PR TITLE
implemented landing page.

### PR DESCRIPTION
**Pull request by SKOLZ**
## Summary

implemented landing
## Screenshots

screen shot 2016-03-07 at 09 53 02
screen shot 2016-03-07 at 09 53 10
screen shot 2016-03-07 at 09 53 14
## Improvements
- select icons for features
- select backgrounds for features
- change youtube video for the wolox jingle

### CODE REVIEW

**Comment by SKOLZ** [comment]
@epintos check this pr too

File: app/views/landing/index.html.slim [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      = image_tag 'feature_icon_3.png', class: 'feature-icon'
+      p.p-feature-description
+        | Integrate CodeStats with your pull requests and avoid undesired surprises during mantainance
+.about-section
+  .about-info
+    .headline-container
+      h2.h2-subtitle
+        | Developed by
+      = image_tag "wolox_logo.png", class: 'wolox-logo'
+    p.p-description
```
Comment (epintos) [inline app/views/landing/index.html.slim]: Add yourself :p

File: app/views/landing/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      | Features
+  .features
+    .feature
+      = image_tag 'feature_icon_1.png', class: 'feature-icon'
+      p.p-feature-description
+        | Collect any metric you want from any branch you want
+    .feature
+      = image_tag 'feature_icon_2.png', class: 'feature-icon'
+      p.p-feature-description
+        | Select the metrics that are most important to you and they will be shown in the main page
```
Comment (epintos) [inline app/views/landing/index.html.slim]: This should be with I18n

File: app/controllers/landing_controller.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```
Comment (epintos) [inline app/controllers/landing_controller.rb]: Why dont you extend this from ActionController::Base and leave everything like it was before?

**Comment by epintos** [comment]
Amazing job! A few comments only

File: app/views/landing/index.html.slim (hunk 1 above)
Comment (SKOLZ) [inline app/views/landing/index.html.slim]: :relaxed:

File: app/controllers/landing_controller.rb (hunk 3 above)
Comment (SKOLZ) [inline app/controllers/landing_controller.rb]: i'll do it that way.

**Comment by SKOLZ** [comment]
@epintos i corrected those things

**Comment by epintos** [comment]
:+1:

**Pull request merged**